src/extension.ts

```diff
@@ -48,7 +48,16 @@ class OpenController implements vscode.Disposable {
     this._disposable.dispose();
   }
 
-  private open(uri: vscode.Uri | undefined): void {
+  private open(uri: vscode.Uri | undefined, uris: vscode.Uri[] | undefined): void {
```

Comment:
Thanks for reviewing.

Do you mean removing uri parameter and use only uris parameter like as below
```javascript
  this.open(uris);
  ---
  private open(uris: vscode.Uri[] | undefined): void {
```
or combining two parameters into one as tuple like as below.
```javascript
  this.open(uri, uris);
  ---
  private open(uriArgs: [vscode.Uri | undefined, vscode.Uri[] | undefined]): void {
```

If the former, it might be good but I am still not sure callback function always has uris parameter when it has uri parameter. Because it's not described clearly in reference.
What I found is only this as below.
> Note 1: When a command is invoked from a (context) menu, VS Code tries to infer the currently selected resource and passes that as a parameter when invoking the command. For instance, a menu item inside the Explorer is passed the URI of the selected resource and a menu item inside an editor is passed the URI of the document.
> [vscode api reference](https://code.visualstudio.com/api/references/contribution-points#contributes.menus)


If the latter, I don't understand why.

If the other, could you explain me more detail.